src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -11736,7 +11736,7 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
         // Key outputs of this subroutine
         this->CompActSpeed = max(CompSpdActual, 0.0);
         this->Ncomp = max(Ncomp, 0.0) / this->EffCompInverter;
-        this->OUFanPower = this->RatedOUFanPower;
+        this->OUFanPower = this->RatedOUFanPower * CyclingRatio;
```

Comment:
This seems like a reasonable change IF the system uses cycling fan. For constant fan I don't think you want to do this.

Reply:
Oh, wait. this is the outdoor unit fan. Yes, I think this is reasonable that the entire system would cycle when below min part load ratio.